README.md

```diff
@@ -11,4 +11,34 @@ the folder 3rdparty.
 
 Screenshots
 -----------
-![Alan Wakes American Nightmare 1](screenshots/awan1.png)
\ No newline at end of file
+![Alan Wakes American Nightmare 1](screenshots/awan1.png)
+
+
+Building and running on Linux
+--------
+
+Make sure to not include the `$` when running these. It signifies that the commands should be run as your user, not root.
+
+```bash
+$ git clone https://github.com/OpenAWE-Project/OpenAWE.git --recurse-submodules
+$ cd OpenAWE
+$ mkdir build
+$ cd build
+```
+
+The following cmake invocation will fail if you don't have all necessary dependencies installed on your host system.
+Acquiring these is distribution-dependent, but generally you'll want the -devel or -dev packages of what it says
+is missing.
+
+```
+$ cmake ..
+$ make
+```
+
+Assuming these steps went well, you should now have built OpenAWE. To run OpenAWE, do something like this. Note that
+you need to replace the path with the actual path to your Alan Wake installation, and that only
```

Comment:
Would be nice to detail, that the data directory in the installation directory is needed